repository_service_tuf/cli/admin/metadata.py

```diff
@@ -103,6 +108,19 @@
 Note: there can be only one online key at a time.
 """
 
+METADATA_SIGNING = """
+# Metadata Signing
+
+Metadata signing allows sending signature of pending Repository Service for TUF
+(RSTUF) role metadata.
+
+It retrieves the pending metadata from the RSTUF API.
+Select the metadata role pending signature and the private key to load.
+
+After loading the key it will sign the role metadata and send the request to
+the RSTUF API with the signature.
```

Comment:
We can remove the last two items.
> - signs the chosen metadata role
> - sends a request to RSTUF API with the new signature

It is duplicated in the sentence below

> After loading the key it will sign the role metadata and send the request to the RSTUF API with the signature.